### PR TITLE
Some e2e test tweaks and small fixes

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
hey @njogz . Could you please have a look? Thanks!

File: tests/e2e/sentinel/transitions/mark_for_outbound.js [hunk 1]
```diff
@@ -75,12 +75,12 @@ const wipeTasks = () => getTasks()
...
 describe('mark_for_outbound', () => {
-  afterEach(done => Promise.all([utils.revertSettings(), wipeTasks()]).then(done));
-  afterAll(done => utils.revertDb().then(done));
+  afterEach(() => Promise.all([utils.revertSettings(true), wipeTasks()]));
+  afterAll(() => utils.revertDb());
 
   describe('when external server is up', () => {
-    beforeEach(done => startMockApp().then(done));
-    afterEach(done => { stopMockApp(); done(); });
+    beforeEach(() => startMockApp());
```
Comment (njogz) [inline tests/e2e/sentinel/transitions/mark_for_outbound.js:82]: Make the same change on line 429 of this file

File: tests/e2e/sentinel/transitions/mark_for_outbound.js [hunk 2]
```diff
@@ -75,12 +75,12 @@ const wipeTasks = () => getTasks()
...
+    afterEach(() => void stopMockApp());
```
Comment (njogz) [inline tests/e2e/sentinel/transitions/mark_for_outbound.js:83]: Is `void` needed here given `stopMockApp` does not return anything?

**Review by njogz — CHANGES_REQUESTED** [review]
Good job on this! I have left one suggestion and a question.

File: tests/e2e/sentinel/transitions/mark_for_outbound.js (hunk 2 above)
Comment (dianabarsan) [reply tests/e2e/sentinel/transitions/mark_for_outbound.js]: nope, it isn't. I think the initial idea was that we shouldn't wait for the mock server to close but then additional stuff was added to that function.

File: tests/e2e/sentinel/transitions/mark_for_outbound.js (hunk 1 above)
Comment (dianabarsan) [reply tests/e2e/sentinel/transitions/mark_for_outbound.js:82]: done.

**Review by njogz — APPROVED** [review]
Looks good.

**Pull request merged**